File: Bronze/Bronze_4/C_files/Code/Average_Score_10039.c
#include <stdio.h>

int main()
{
    int score[5], sum = 0;

    for (int i = 0; i < 5; i++) {
        scanf("%d", &score[i]);
        if (score[i] < 40)
            sum += 40;
        else
            sum += score[i];
    }
    
    printf("%d", sum / 5);

    return 0;
}

File: Bronze/Bronze_4/Java_files/Code/Average_Score_10039.java
import java.util.Scanner;

public class Average_Score_10039 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] score = new int[5];
        int sum = 0;
        for (int i = 0; i < 5; i++) {
            score[i] = sc.nextInt();
            if (score[i] < 40)
                sum += 40;
            else
                sum += score[i];
        }
        System.out.println(sum / 5);
    }
}
